rules/order.js

```diff
@@ -0,0 +1,101 @@
+module.exports = {
+  'order/properties-order': [
+    {
+      order: 'strict',
+      properties: [
+        'content',
+      ],
+    },
+    {
+      order: 'strict',
+      properties: [
+        'position',
+        'z-index',
+        'top',
+        'bottom',
+        'left',
+        'right',
+      ],
+    },
+    {
+      order: 'strict',
+      properties: [
+        'transform',
+        'transform-origin',
+      ],
+    },
+    {
+      order: 'strict',
+      properties: [
+        'box-sizing',
+        'display',
+        'overflow',
+        'flex',
+        'flex-grow',
+        'flex-shrink',
+        'flex-basis',
+        'order',
+        'align-self',
+        'flex-direction',
+        'flex-wrap',
+        'justify-content',
+        'align-items',
+        'align-content',
+        'height',
+        'width',
+        'margin',
+        'margin-top',
```

Comment:
Apologies, I should have been a bit more specific.

Shorthand notation for `margin`, `padding`, `border` etc. is: `property: <top> <right> <bottom> <left>`. This means our mental model around reading margin/padding/border code is set the same way. However, reading top/bottom/left/right is counter to this mental model and could make reading/scanning code a tiny bit harder. 

Does this make sense?